ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите 1 катет: ");
+            double a = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите 2 катет: ");
+            double b = int.Parse(Console.ReadLine());
+            double S = (a + b) / 2;
```

Comment:
Формула вычисления площади неверная
```suggestion
            double S = (a * b) / 2;
```